### DISCUSSION TITLE
How to use the bootstrapping functionality in PCMCI/PCMCIplus?

### DISCUSSION THREAD

**Discussion by f-arab** [jakobrunge/tigramite#263] [Q&A] [answered]
Hi everyone,

We want to do bootstrapping while running PCMCI or PCMCIplus methods on our data. We see that inside the code there is functionality for it but we are not exactly sure how to use that! Can anyone please point us to a quick way of using bootstrap functionality in PCMCI/ PCMCIplus methods?

Thanks

**Comment by jakobrunge** [comment, marked as answer]
This is now implemented, see https://github.com/jakobrunge/tigramite/blob/master/tutorials/causal_discovery/tigramite_tutorial_bootstrap_aggregation.ipynb
